css/style.css

```diff
@@ -1,9 +1,379 @@
+  font-size: 37px;
+}
+
+.red-like-color {
+  color: #ec5242;
+}
+
+.article-paragraph {
+  border: 2px solid #fff;
+  line-height: 27px;
+  margin: 34px 0;
+  padding: 4px;
+}
+
+.date-paragraph {
+  font-size: 21px;
+  font-weight: bolder;
+}
+
+.main-program-div {
+  background-color: #000;
+  color: #fff;
+  margin-top: 46px;
+  background-image: url(../images/bacground-image.jpg);
+  background-repeat: round;
+  background-size: contain;
+  width: 100%;
+}
+
+.paragraph-border {
+  border-bottom: 2px solid #ec5242;
+  width: 28px;
+  margin: 10px auto;
+}
+
+.main-program-ul {
+  background-color: #272a31;
+  margin: 4px 22px;
+  align-items: center;
+}
+
+.main-program-ul-li {
+  padding: 34px 8px;
+}
+
+.lecture-image {
+  height: 40px;
+}
+
+.topic {
+  font-size: 19px;
+  color: #ec5242;
+}
+
+.main-program-button-div {
+  padding-bottom: 50px;
+}
+
+.join-button {
+  background-color: #ec5242;
+  opacity: 1;
+  padding: 5%;
+  margin: 23px auto;
+  color: #fff;
+  font-size: 17px;
+  display: block;
+}
+
+.join-button:hover {
+  opacity: 0.8;
+}
+
+.see-program {
+  text-decoration: #fff underline;
+  font-size: 20px;
+  margin: 24px 0;
+}
+
+.join-anchore:hover {
+  opacity: 0.8;
+}
+
+.all-wrappers {
+  margin: 25px 0;
+}
+
+.featured-speakers-ul {
+  margin: 24px 0;
+}
+
+.featured-speaker-image {
+  height: 100px;
+}
+
+.lato-font {
+  font-family: 'Lato', sans-serif;
+}
+
+.featured-speakers-li {
+  margin: 0 12px;
+}
+
+.featured-speakers-ul-li {
+  padding: 34px 0;
+}
+
+.featured-speaker-paragraph {
+  line-height: 23px;
+}
+
+.about-body {
+  color: #272a31;
 }
 
-.header {
-  color: wheat;
-  font-family: Metal Mania, "Times New Roman", Times, serif;
+.about-paragraph,
+.about-paragraph-2,
+.about-second-paragraph {
+  line-height: 24px;
+  margin: 14px 0;
+}
+
+.about-paragraph,
+.commmon-goals-div {
+  background-color: #fff;
+}
+
+.about-logo {
+  margin: 45px;
+}
+
+.summit-title {
+  font-size: 23px;
+  margin: 8px;
+}
+
+.about-paragraph-2 {
+  margin-top: 200px;
+}
+
+.summit-first-div {
+  background-color: #fff;
+}
+
+.summit-image-div {
+  background-image: url('../images/summit-1.jpg');
+}
+
+.summit-image-div-2 {
+  background-image: url('../images/refugee-summit-1.jpg');
+}
+
+.summit-img-styles {
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 80%;
+  height: 216px;
+  margin: 24px auto;
+}
+
+.summit-image-figcaption {
+  background-color: #ec5242;
+  height: 216px;
+  opacity: 0.5;
   text-align: center;
+  padding: 86px 0;
+  color: #fff;
+  font-weight: bolder;
+  font-size: 19px;
+}
+
+.partner {
+  background-color: #272a31;
+}
+
+.social-media-image {
+  margin: 14px 24px;
+}
+
+.footer-image {
+  height: 72px;
+  margin: 24px;
+}
+
+.footer-2-ul-li {
+  width: 85%;
+}
+
+.mozilla {
+  font-size: 20px;
+  color: #d3d3d3;
+}
+
+.daumakakau {
+  font-size: 18px;
+  color: #555;
+}
+
+.footer-image-title {
+  font-size: 17px;
+}
+
+.footer-ul {
+  width: 90%;
+  margin: 4px auto;
+}

```

Comment:
- [ ] Kindly to match the previous review changes, you can add `padding-bottom: 40px;` to make your design match the original design.

Reply:
Addressed ✔